app/views/accounts/_funding_details.html.erb

```diff
@@ -28,14 +28,14 @@
 
         <% scholarship_funding_row(application) %>
 
-        <% if Flipper.enabled?(:targeted_support_funding) %>
+        <% if Flipper.enabled?(:targeted_support_funding) || application.created_at < Time.zone.local(2024, 4, 2) %>
```

Comment:
It would be lovely to have one source of `magic numbers`. How about file `config/initializers/constants.rb` and there:
```ruby
APRIL_2024_CUTOFF_DATE = Time.zone.local(2024, 4, 2)
CURRENT_CUTOFF_DATE = APRIL_2024_CUTOFF_DATE  # in case we need generic cutoff
```
This way we will have all constants in one page. I don't think we need to worry about namespacing them as the app is rather small.